packages/calcite-components/conventions/Accessibility/AccessibilityDesign.md

```diff
@@ -1,12 +1,57 @@
 # Design Quick Start Guide
 
-1. **[Color](#1-color)**: [Color contrast](#color-contrast) | [Color and state](#color--state)
-2. **[Typography](#2-typography)**: [Text styling](#text-styling) | [Text resizing](#text-resizing)
-3. **[Information hierarchy](#3-information-hierarchy)**: [Semantic structure](#semantic-structure) | [Headings](#headings) | [Links](#links) | [Navigation](#navigation) | [Focus order](#focus-order)
-4. **[Forms](#4-forms)**: [Labels](#labels) | [Grouping controls](#grouping-controls) | [Form fields](#form-fields) | [Buttons and disabled buttons](#buttons) | [User notifications](#user-notifications) | [Touch zones](#touch-zones)
-5. **[Images](#5-images)**: [Background images](#background-images) | [Icons](#icons)
-6. **[Animation](#6-animation)**
-7. **[Writing and labels](#7-writing-and-labels)**: [Error messaging](#error-messaging)
+<!-- prettier-ignore-start -->
```

Comment:
Are these disabled to avoid linting fixes from messing up the TOC? If so, could we add an explainer to these?

Reply:
Yeah exactly. I thought the `table of contents auto generated by https://github.com/artempyanykh/marksman` was the explainer, but I'll add more context.